src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolderDescriptor.java

```diff
@@ -41,6 +41,40 @@ public String getDisplayName() {
     }
 
     /**
+     * Needed if it wants Folders are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     * TODO: Replace to {@code NestedProjectsCategory.ID}
```

Comment:
Well I think once we have a 2.x dependency, we can just move that category definition here.